src/app/betting/feature/place-bet-page.component.ts

```diff
@@ -66,13 +77,111 @@ import { BetStatus } from '../../shared/util/supabase-helpers';
       />
     </label>
 
-    <!-- Bet terms -->
+    <!-- Bet Type Dropdown -->
+    <!-- eslint-disable-next-line -->
```

Comment:
Since we're using this so much in our project, can you remove all instances of `<!-- eslint-disable-next-line -->`, and turn off `@angular-eslint/template/label-has-associated-control` in `.eslintrc.json`?